api/HTMLLIElement.json

```diff
@@ -48,6 +48,100 @@
           "standard_track": true,
           "deprecated": false
         }
+      },
+      "type": {
```

Comment:
Both "type" and "value" are reflected attributes, can you update the content attribute data too?